Model/User.php

```diff
@@ -536,13 +677,17 @@ public function setRoles(array $roles)
     /**
      * Gets the groups granted to the user.
      *
-     * @return Collection
+     * @return ArrayCollection
      */
     public function getGroups()
     {
         return $this->groups ?: $this->groups = new ArrayCollection();
     }
 
+    /**
+     * 
+     * @return string[]
+     */
```

Comment:
All these methods are already documented in the interface. Omitting the phpdoc is done on purpose